pinot-core/src/main/java/org/apache/pinot/core/plan/FilterPlanNode.java

```diff
@@ -56,6 +59,10 @@
 
 
 public class FilterPlanNode implements PlanNode {
+
+  private static final Set<String> CAN_APPLY_H3_INCLUSION_INDEX_FUNCTION_NAMES =
+      ImmutableSet.of("st_within", "stwithin", "st_contains", "stcontains");
```

Comment:
(minor) No need to check `st_within` and `st_contains` because the function name is already canonicalized (we keep 2 function names for `st_distance` for backward compatibility). We may directly use the value equals check into the `canApplyH3IndexForInclusionCheck()` instead of using a set for 2 values.

Reply:
sure.